README.md

```diff
@@ -14,5 +11,32 @@ The scripts install the following:
 There are also scripts for installing software individually. Software that currently has individual install scripts:
 - [TEMOA](https://github.com/TemoaProject/temoa)
 
+# Requirements
+ - The user must fork all of the listed software tools (unless otherwise specified) listed below for the scripts to run successfully. 
+ - The user must have [conda]() installed. Currently, the software only works if with anaconda. The `anaconda3` must be located in `home`. I recommend using Sam Dotson's script [ubuntu-post-installer](https://github.com/samgdotson/ubuntu-post-installer) to do this, as well as set up a few other useful pieces of software on ubuntu)
```

Comment:
Whoops! Missed that I didn't include a link to conda. This is why you should review your code.